synapse/lib/stormtypes.py

```diff
@@ -6204,6 +6208,18 @@ async def _methViewMerge(self, force=False):
         todo = s_common.todo('merge', useriden=useriden, force=force)
         await self.runt.dyncall(viewiden, todo)
 
+    async def _methWipeLayer(self):
+        '''
+        Delete nodes and nodedata from the view's write layer.
+        '''
+        useriden = self.runt.user.iden
+        viewiden = self.valu.get('iden')
+        view = self.runt.snap.core.getView(viewiden)
+
+        # ensure the user may make *any* node edits
+        self.runt.layerConfirm(('node',))
```

Comment:
Two other options here:
- Confirm the various del perms
- Similar to merge() go through the edits and confirm perms on each

Reply:
We should confirm each perm, since someone could have a explicit deny perm preventing the particular edit from occuring.

Reply:
I've rethought this and changed my mind.  Lets go through all the edits and check individual perms ( with an obvious cache ) in advance of starting.